web/src/user_group_edit.ts

```diff
@@ -57,13 +59,25 @@ import * as user_groups from "./user_groups.ts";
 import type {UserGroup} from "./user_groups.ts";
 import * as user_profile from "./user_profile.ts";
 import * as util from "./util.ts";
+import * as views_util from "./views_util.ts";
 
 type ActiveData = {
     $row: JQuery | undefined;
     id: number | undefined;
     $tabs: JQuery;
 };
 
+let filters_dropdown_widget;
+export const FILTERS = {
+    ACTIVE_AND_DEACTIVATED_GROUPS: $t({defaultMessage: "Active and deactivated"}),
+    ACTIVE_GROUPS: $t({defaultMessage: "Active groups"}),
+    DEACTIVATED_GROUPS: $t({defaultMessage: "Deactivated groups"}),
+};
+const ls = localstorage();
+export const user_group_visibility_key = "visible_user_groups";
+const initial_visible_groups = z.string().optional().parse(ls.get(user_group_visibility_key));
+export let selected_dropdown_filters =
```

Comment:
Is there any need for this variable? Current selected filter can be used from `dropdown_widget.value()` and `default_id` for DropdownWidget from `initial_visible_groups`.